raiden-ts/src/messages/utils.ts

```diff
@@ -285,14 +285,14 @@ export function getMessageSigner(message: Signed<Message>): Address {
 }
 
 /**
- * Get the SignedBalanceProof associated with an EnvelopeMessage
+ * Get the signed BalanceProof associated with an EnvelopeMessage
  *
  * @param message - Signed EnvelopeMessage
- * @returns SignedBalanceProof object for message
+ * @returns Signed BalanceProof object for message
  */
 export function getBalanceProofFromEnvelopeMessage(
   message: Signed<EnvelopeMessage>,
-): SignedBalanceProof {
+): Signed<BalanceProof> {
```

Comment:
Ok I got my answer here :+1: The review interface is really inconvinient for such big PRs you cannot easily get the context.